mmrazor/models/mutables/mutable_channel/__init__.py

```diff
@@ -1,8 +1,31 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+r"""This module defines MutableChannels.
+
+----------------------------------------------------------base_mutable_channel.py
+BaseMutableChannel
+|                            \
+----------------------------------------------------------mutable_channel_container.py
+MutableChannelContainer        \
+----------------------------------------------------------other files
+                                 \   other MutableChannels
+
+MutableChannel are mainly used in DynamicOps. It helps DynamicOps to deal
+with mutable number of channels.
+"""
+from .base_mutable_channel import BaseMutableChannel
+from .groups import (MUTABLECHANNELGROUP, MutableChannelGroup,
+                     SequentialChannelGroup)
 from .mutable_channel import MutableChannel
+from .mutable_channel_container import MutableChannelContainer
 from .one_shot_mutable_channel import OneShotMutableChannel
+from .sequential_mutable_channel import SquentialMutableChannel
+from .simple_mutable_channel import SimpleMutableChannel
 from .slimmable_mutable_channel import SlimmableMutableChannel
 
 __all__ = [
-    'OneShotMutableChannel', 'SlimmableMutableChannel', 'MutableChannel'
+    'SimpleMutableChannel', 'MutableChannelGroup', 'OneShotChannelGroup',
+    'BaseMutableChannel', 'MutableChannelContainer', 'StackMutableChannel',
+    'MUTABLECHANNELGROUP', 'MutableChannel', 'OneShotMutableChannel',
```

Comment:
`MUTABLECHANNELGROUP` just is a typa var, why add it here?  Will it be used in other modules?

Reply:
It's used in Mutator, or it can be declared  in Mutator？